content/docs/sidebar.json

```diff
@@ -5,6 +5,29 @@
     "source": "index.md",
     "icon": "house"
   },
+  {
+    "label": "CML",
+    "slug": "cml",
+    "source": "cml/index.md",
+    "icon": "cml",
+    "children": [
+      "usage",
+      "get-started",
```

Comment:
Also maybe Get Started before Usage?

Reply:
good idea- putting "Usage" first was a convention I picked up GitHub docs for releasing actions (they recommend a "Usage" section, but not "Getting Started" or a tutorial necessarily).